assets/js/services/adverts.service.js

```diff
@@ -21,16 +21,27 @@ function Adverts ($http, $rootScope) {
   }
 
   function fetch () {
-    let advertsFeed = $rootScope.rootURL + 'adverts_feed?wallet_version=3';
+    if (!$rootScope.apiDomain.endsWith('blockchain.info/')) {
+      return;
+    }
+
+    let advertsFeed = $rootScope.apiDomain + 'ads/get?wallet=true&n=2';
     $http.get(advertsFeed)
-      .success(data => {
-        let adverts = data.partners.home_buttons.splice(0);
-        service.ads.push(randFromArray(adverts));
-        service.ads.push(randFromArray(adverts));
-      });
-  }
+      .success(function (data) {
+        data.forEach(function (ad) {
+          var a = document.createElement('a');
```

Comment:
This is too jQuery-ish. @jtormey and @plondon should be able to help you do this the Angular Way. It would probably involve an `ng-repeat` in the template.